src/libraries/Microsoft.Bcl.Cryptography/tests/X509Certificates/X509CertificateKeyAccessorsTests.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography.SLHDsa.Tests;
+using System.Security.Cryptography.X509Certificates;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests
+{
+    public static class X509CertificateKeyAccessorsTests
+    {
+        [ConditionalFact(typeof(SlhDsa), nameof(SlhDsa.IsSupported))]
+        public static void GetPublicKey()
+        {
+#if NET10_0_OR_GREATER
+            string certPem = PemEncoding.WriteString("CERTIFICATE", SlhDsaTestData.IetfSlhDsaSha2_128sCertificate);
+            using (X509Certificate2 cert = X509Certificate2.CreateFromPem(certPem))
+            using (SlhDsa? certKey = X509CertificateKeyAccessors.GetSlhDsaPublicKey(cert)) // Use extension method
+            {
+                Assert.NotNull(certKey);
+                AssertExtensions.SequenceEqual(SlhDsaTestData.IetfSlhDsaSha2_128sPublicKeyValue, certKey.ExportSlhDsaPublicKey());
+            }
+#else
+            // Unix: OpenSsl accessors not supported downlevel
+            // Windows: SlhDsa currently not supported so test won't execute
+            Assert.Throws<PlatformNotSupportedException>(() => X509CertificateKeyAccessors.GetSlhDsaPrivateKey(null));
```

Comment:
Will this assert even be hit because it's a `ConditionalFact`?

* Windows returns `false` right now....
* OOB Linux should return `false` right now...

So I am failing to see when this `#else` will be hit.